limesurvey/limesurvey.py

```diff
@@ -208,7 +208,7 @@ def setup_student_view_survey(self, user, anonymous_user_id):
         if not limesurvey_url:
             raise MisconfiguredLimeSurveyService("LIMESURVEY_URL is not set in your service configurations.")
 
-        self.survey_url = f"{limesurvey_url}/{self.survey_id}"
+        self.survey_url = f"{limesurvey_url}/index.php/{self.survey_id}"
```

Comment:
[Couriose] Can this work without `.php`? it which means like below:

`{limesurvey_url}/index/{self.survey_id}`

Reply:
No, it doesn't. For older installations the path includes the `.php`